packages/studio-base/src/panels/RawMessages/types.ts

```diff
@@ -22,4 +22,6 @@ export type RawMessagesPanelConfig = {
 export const Constants = {
   CUSTOM_METHOD: "custom",
   PREV_MSG_METHOD: "previous message",
+  FONT_SIZE_OPTIONS: [8, 9, 10, 10.5, 11, 12, 14, 18, 20, 22, 24, 28, 32],
```

Comment:
I would suggest updating size values to be more consistent with typography metrics 
```suggestion
  FONT_SIZE_OPTIONS: [8, 9, 10, 11, 12, 14, 16, 18, 24, 30, 36, 48, 60, 72],
```